scripts/goldpolish-target-post-processing.py

```diff
@@ -0,0 +1,109 @@
+"""Inserts polished gap sequences back into the original assembly"""
+# imports
+import argparse
+import btllib
+
+
+def parse_args():
+    """parses args passed through command line"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-f", "--fasta", help="target file in fasta format", type=str, required=True
+    )
+    parser.add_argument(
+        "-g",
+        "--gaps",
+        help="polished gaps and flanks in fasta format",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "-o",
+        "--output",
+        help="name of output file",
+        type=str,
+        required=True
+    )
+    return parser.parse_args()
+
+
+def insert_seq(gap_coords, sequence):
+    """inserts gap_sequences into contig at the given coordinates"""
+    start = 0
+    updated_seq = []
+
+    # reading gap names to get coordinates
+    for gap in gap_coords:
+        gap_start = gap[1]
+        gap_end = gap[2]
+        gap_sequence = gap[3]
+
+        updated_seq.append(sequence[start : int(gap_start)])
+        updated_seq.append(gap_sequence)
+        start = int(gap_end)
+
+    # just to check
+    if start < len(sequence) - 1:
+        updated_seq.append(sequence[int(start):])
+    return "".join(updated_seq)
+
+
+def make_seq_dict(gaps):
```

Comment:
```suggestion
def make_seq_dict(gaps):
"Returns a dictionary containing the gap sequence information"
```
Just good to have a short function descriptor